### ISSUE TITLE
Setting up γ for ImageNet 256

### ISSUE THREAD

**Issue by Peterande** [brownvc/R3GAN#5]
![image](https://github.com/user-attachments/assets/9350f5a8-78e4-40c6-851d-207b82e1c50e)

Hello, great work, can you give me some advice on setting up γ? I can't summarize the rules well from the table, for example, the difference between FFHQ γ values of different resolutions is too large

**Comment by lanqingxi** [comment]
1. I have also noticed this problem, in my application, when using FFHQ-256 configured gamma values and strategies, the loss of the discriminator in the pre-training period leads to abnormally large losses, even though lowering that configuration to 15, 1.5, again leads to large losses in the discriminator.
2. my discriminator is based on a pre-trained caformer, is this due to the use of pre-trained weights?

**Comment by Blade6570** [comment]
Same problem. I have also tried with bigger version of PatchGAN and loss increases abnormally.
